components/Layout.tsx

```diff
@@ -1,10 +1,32 @@
 import { inter } from "@styles/fonts";
+import Link from "next/link";
 import Footer from "./Footer";
 import { Header } from "./Header";
 
+function Banner() {
+  return (
+    <div style={{ backgroundColor: "#EEEAFF" }} className="w-full py-3 sm:py-5">
```

Comment:
Using a very light shadow of purple would probably have a similar/ the same effect so instead of setting background color using styles just use bg-purple-100